### PR TITLE
[14.0][FIX] website_cookiefirst: Bad inherited view

### PR DESCRIPTION
**Pull request by matteoopenf**
load could not load template
ValueError: L'elemento '' non può esser localizzato nella vista genitore https://github.com/OCA/website/issues/968

### CODE REVIEW

**Review by pedrobaeza — COMMENTED** [review]
I think you should change the inherited template:

https://github.com/odoo/odoo/blob/9cc56fc3d3cdb43e393df0975e10192c3b10f455/addons/website/views/website_templates.xml#L179

**Comment by matteoopenf** [comment]
> I think you should change the inherited template:
> 
> https://github.com/odoo/odoo/blob/9cc56fc3d3cdb43e393df0975e10192c3b10f455/addons/website/views/website_templates.xml#L179

I check and I will update.
@pedrobaeza  thank for quick response

**Comment by matteoopenf** [comment]
> I think you should change the inherited template:
> 
> https://github.com/odoo/odoo/blob/9cc56fc3d3cdb43e393df0975e10192c3b10f455/addons/website/views/website_templates.xml#L179

confirm I updated the Pr, thanks again

File: website_cookiefirst/views/portal_template.xml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8" ?>
 <odoo>
-    <template id="footer_custom" inherit_id="website.footer_custom">
+    <template id="layout" inherit_id="portal.frontend_layout">
```
Comment (pedrobaeza) [inline website_cookiefirst/views/portal_template.xml]: For not losing the change when installing website, it should be:

```suggestion
    <template id="layout" inherit_id="website.layout">
```

Comment (pedrobaeza) [reply website_cookiefirst/views/portal_template.xml]: Ah, and better to keep the same XML-ID for avoiding problems on migrations:

```suggestion
    <template id="footer_custom" inherit_id="website.layout">
```

Comment (matteoopenf) [reply website_cookiefirst/views/portal_template.xml]: if I do that I have the previus error

Comment (pedrobaeza) [reply website_cookiefirst/views/portal_template.xml]: Try with a less strict xpath `//footer//span...` instead of `//footer//div/span`

**Comment by matteoopenf** [comment]
I squash the commit

**Comment by matteoopenf** [comment]
@pedrobaeza  I See when I squash the commit I lost the coauthor of the commit, is a problem for you?

**Comment by pedrobaeza** [comment]
Don't worry. No problem in this authorship. It's not a big deal in this case :smiley:

**Comment by matteoopenf** [comment]
> Don't worry. No problem in this authorship. It's not a big deal in this case smiley

I readd the coauthor.
Thanks again for the help and suggestion and the patience

File: website_cookiefirst/views/portal_template.xml [hunk 2]
```diff
@@ -1,10 +1,7 @@
 <?xml version="1.0" encoding="utf-8" ?>
 <odoo>
-    <template id="footer_custom" inherit_id="website.footer_custom">
-        <xpath
-            expr="//footer//div/span[hasclass('o_footer_copyright_name')]"
-            position="after"
-        >
+    <template id="footer_custom" inherit_id="website.layout">
+        <xpath expr="//footer//span/.." position="after">
```
Comment (pedrobaeza) [inline website_cookiefirst/views/portal_template.xml]: When I put the `...` (3 dots), it was for not writing the rest of the expression, but I think you still need to use the `hasclass` part to select the exact element.

Comment (matteoopenf) [reply website_cookiefirst/views/portal_template.xml]: ok I check I will come back

Comment (matteoopenf) [reply website_cookiefirst/views/portal_template.xml]: i specify the xpath now

**Review by pedrobaeza — APPROVED** [review]

**Review by lorenzoallegrucci — APPROVED** [review]

**Comment by pedrobaeza** [comment]
/ocabot merge minor

Please check if needed in upper versions.

**Pull request merged**